vcd/resource_vcd_edgegateway.go

```diff
@@ -113,6 +114,25 @@ func resourceVcdEdgeGateway() *schema.Resource {
 				Description: "Log level. One of 'emergency', 'alert', 'critical', 'error', " +
 					"'warning', 'notice', 'info', 'debug'. ('info' by default)",
 			},
+			"fw_enabled": &schema.Schema{
```

Comment:
Did we have vote between this variation ?:)
```
firewall{
 enabled = true
 default_logging_enabled=true
 default_rule_action = accept
}
```
Also accept and deny could be boolean with meaningful property name.

Reply:
@lvirbalas did we already have guideline for such cases ?:)

Reply:
There was no guideline, but after we learned all the nasty things about `Set` you have hit with the VDC data source, I think we became "fine" with flattened parameter names like here.